Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -170,10 +223,26 @@ class TextureCacheBase
   TCacheEntry* Load(const u32 stage);
   static void InvalidateAllBindPoints() { valid_bind_points.reset(); }
   static bool IsValidBindPoint(u32 i) { return valid_bind_points.test(i); }
-  void BindTextures();
+  TCacheEntry* GetTexture(u32 address, u32 width, u32 height, const TextureFormat texformat,
+                          const int textureCacheSafetyColorSampleSize, u32 tlutaddr = 0,
```

Comment:
`snake_case`, also `texformat`, `tlutaddr` and `tlutfmt` aren't consistent in their underscoring with the rest of the parameters. `texformat` and `textureCacheSafetyColorSampleSize` don't really need to be const in the header file. Ditto for any occurrences below.

Reply:
Good call, I'll take a look.  This was me copy-pasting from the previous implementation.  Sorry about that.

Reply:
Hmm, I put these functions in the middle of some other ones.  There's virtual functions everywhere, which likely falls into the "clean up later" PR.